src/Shared/Recruit.Vacancies.Client/Domain/Entities/Vacancy.cs

```diff
@@ -101,5 +101,10 @@ public class Vacancy
         public bool CanSendForReview => Status == VacancyStatus.Submitted && IsDeleted == false;
 
         public bool IsDisabilityConfident => DisabilityConfident == DisabilityConfident.Yes;
+
+        /// <summary>
+        /// We can extend the ClosingDate and StartDate for Live vacancies that have not been deleted
+        /// </summary>
+        public bool CanExtendStartAndClosingDates => Status == VacancyStatus.Live && IsDeleted == false;
```

Comment:
Can a live vacancy be soft deleted? Would it not just be closed, otherwise we will have some lost orphan published vacancy known in the v2 system and published on FAA. This confuses the reasoning of the vacancy state/domain model.

Reply:
I agree we would never delete a live vacancy but we should always honour the IsDeleted flag.  Conceptually you should never be able to change any data on something that has been deleted.  This check is just belt-and-braces.